### PR TITLE
Making argument forwarding tests run with dotnet test

### PR DESCRIPTION
**Pull request by livarcocc**
Making argument forwarding tests run with dotnet test and removing the its special test scripts.

### CODE REVIEW

**Comment by livarcocc** [comment]
cc @brthor @piotrpMSFT

File: test/ArgumentForwardingTests/project.json [hunk 1]
```diff
@@ -5,22 +5,23 @@
...
     "frameworks": {
         "dnxcore50": {
             "imports": "portable-net45+win8"
         }
-    }
+    },
+    
+    "testRunner": "xunit",
+    
+    "scripts": { "precompile": "dotnet build ..\\ArgumentsReflector\\project.json --framework dnxcore50 --output %compile:RuntimeOutputDir%" }
```
Comment (brthor) [inline test/ArgumentForwardingTests/project.json]: Clever solution for the Reflector :+1:

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX Debug Build please
@dotnet-bot Test Windows_NT Debug Build please

**Comment by livarcocc** [comment]
@mmitche I did a dotnet-bot test specific OS and it re-did everything. Have you seen this?

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu Debug Build please

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT Debug Build please

**Comment by mmitche** [comment]
It might be a different set of jobs.  I switched them out for a different set and they would all retrigger.

Sent from my Windows Phone
________________________________

**Comment by brthor** [comment]
:shipit:

**Pull request merged**